.github/ISSUE_TEMPLATE/Release_23.12_FOSS_ComplianceTemplate.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Release Compliance 23.12
```

Comment:
Nachfrage: müsste das hier nicht heissen "Release Compliance [milestone]"?
Oder wolltest Du das Template bewusst so benennen, das es bei 24.03 höchstwahrscheinlich schon wieder anders aussieht?

Reply:
I would also like to use a generic name and update the same file for the next Release Cycle.